app.py

```diff
@@ -30,7 +31,26 @@ def init_app(self):
         Note, this app doesn't register any commands at the moment as all it's functionality is
         provided through it's API.
         """
-        pass
+
+        app = self.import_module("tk_multi_reviewsubmission")
+
+        display_name = self.get_setting("display_name")
+        command_name = display_name.lower()
+        command_name = re.sub("[^0-9a-zA-Z]+", "_", command_name)
+
+        menu_caption = "%s..." % display_name
+        menu_options = {
+            "short_name": command_name,
```

Comment:
I think hardcoding to `send_for_review` is fine here.

Reply:
Any reason why you would need to make it specific? This makes it more difficult to invoke the API from the app if we go this route as you have to know the display name to get to the app.